### DISCUSSION TITLE
How to allow follower for restricted account

### DISCUSSION THREAD

**Discussion by shakks** [tweepy/tweepy#1403] [Questions] [answered]
Is there any API to accept and reject users when they request to follow a restricted account.
Normally I have to do it manually. Is there any API to give or cancel access.

**Comment by Harmon758** [comment, marked as answer]
Unfortunately, this is not currently part of Twitter's API.
See https://twitterdevfeedback.uservoice.com/forums/921790-twitter-developer-labs/suggestions/38545387-accept-or-deny-follower-requests-when-tweets-are.
